### DISCUSSION TITLE
Memento alternative for language server

### DISCUSSION THREAD

**Discussion by gee4vee** [microsoft/vscode-discussions#1160] [Extension Development QnA] [answered]
We developed that relies on `vscode.Memento` for storage and trying to move it to a language server where it will be used and exported to other dependents. Is there an alternative we can use from vscode APIs or a 3rd party lib?

**Comment by isidorn** [comment]
What are you trying to do exactly? Are you a VS Code extension?
Why can't you continue using `vscode.Memento`?

**Comment by burkeholland** [comment, marked as answer]
Closing this as inactive since it's over a month since the last reply here. Feel free to re-open if you feel this is a mistake and can provide more information.
